x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/management/trial_license_complete_tier/upgrade_review_prebuilt_rules.ts

```diff
@@ -711,5 +711,510 @@ export default ({ getService }: FtrProviderContext): void => {
         });
       });
     });
+
+    describe(`scalar array fields`, () => {
+      const getRuleAssetSavedObjects = () => [
+        createRuleAssetSavedObject({
+          rule_id: 'rule-1',
+          version: 1,
+          tags: ['one', 'two', 'three'],
+        }),
+      ];
+
+      describe("when rule field doesn't have an update and has no custom value", () => {
```

Comment:
Can we add the scenarios keys as we did in the other PR? `AAA`, `ABA`, `ABC`, etc...